evaluate/task/symflower-fix.go

```diff
@@ -0,0 +1,50 @@
+package task
+
+import (
+	"context"
+	"time"
+
+	pkgerrors "github.com/pkg/errors"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	"github.com/symflower/eval-dev-quality/language"
+	"github.com/symflower/eval-dev-quality/log"
+	"github.com/symflower/eval-dev-quality/tools"
+	"github.com/symflower/eval-dev-quality/util"
+)
+
+// symflowerFix runs the "symflower fix" command and returns the corresponding assessments.
+func symflowerFix(logger *log.Logger, modelAssessment metrics.Assessments, repositoryPath string, language language.Language) (assessments metrics.Assessments, problems []error, err error) {
+	assessments = metrics.NewAssessments()
+
+	start := time.Now()
+	output, err := util.CommandWithResult(context.Background(), logger, &util.Command{
+		Command: []string{
+			tools.SymflowerPath, "fix",
+			"--language", language.ID(),
+			"--workspace", repositoryPath,
+		},
+
+		Directory: repositoryPath,
+	})
+	if err != nil {
+		return nil, nil, pkgerrors.WithStack(err)
+	}
+	duration := time.Since(start)
+	logger.Printf("with symflower repair: %s", output)
+
+	// The processing time is summed with the one from the model.
+	assessments[metrics.AssessmentKeyProcessingTime] = modelAssessment[metrics.AssessmentKeyProcessingTime] + uint64(duration.Milliseconds())
+
+	coverage, ps, err := language.Execute(logger, repositoryPath)
+	problems = append(problems, ps...)
+	if err != nil {
+		return nil, ps, pkgerrors.WithStack(err)
+	}
+	logger.Printf("with symflower repair: Executes tests with %d coverage objects", coverage)
```

Comment:
I think this should still be part of the task code. Cause symflower fix should not care what happens with its result.